### PR TITLE
Cut release 0.9

### PR DESCRIPTION
**Pull request by dgrisonnet**
## Description

_Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request.
If it fixes a bug or resolves a feature request, be sure to link to that issue._

## Type of change

_What type of changes does your code introduce to the kube-prometheus? Put an `x` in the box that apply._

- [ ] `CHANGE` (fix or feature that would cause existing functionality to not work as expected)
- [ ] `FEATURE` (non-breaking change which adds functionality)
- [ ] `BUGFIX` (non-breaking change which fixes an issue)
- [ ] `ENHANCEMENT` (non-breaking change which improves existing functionality)
- [x] `NONE` (if none of the other choices apply. Example, tooling, build system, CI, docs, etc.)

## Changelog entry

_Please put a one-line changelog entry below. Later this will be copied to the changelog file._

```release-note

```

### CODE REVIEW

**Comment by dgrisonnet** [comment]
I had to revert https://github.com/prometheus-operator/kube-prometheus/commit/f643955034fa6ad4c3311020abd94f4390dd6dee, because this change isn't available in alertmanager release-0.22.

File: jsonnet/kube-prometheus/main.libsonnet [hunk 1]
```diff
@@ -70,7 +70,7 @@ local utils = import './lib/utils.libsonnet';
       image: $.values.common.images.grafana,
       prometheusName: $.values.prometheus.name,
       // TODO(paulfantom) This should be done by iterating over all objects and looking for object.mixin.grafanaDashboards
-      dashboards: $.nodeExporter.mixin.grafanaDashboards + $.prometheus.mixin.grafanaDashboards + $.kubernetesControlPlane.mixin.grafanaDashboards + $.alertmanager.mixin.grafanaDashboards,
```
Comment (paulfantom) [inline jsonnet/kube-prometheus/main.libsonnet:73]: why alertmanager dashboards are removed?

Comment (dgrisonnet) [reply jsonnet/kube-prometheus/main.libsonnet]: fixed it by including the upstream commit that adds the dashboards.

**Comment by simonpasquier** [comment]
lgtm

**Pull request merged**